en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-configuration.md

```diff
@@ -0,0 +1,111 @@
+# Amazon Lambda Configuration
+
+The AmazonLambda Connector allows you to access the REST API of [Amazon Web Service Lambda (AWS Lambda)](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html), which lets you run code without provisioning or managing servers. With Lambda, you can run code for virtually any type of application or backend service - all with zero administration. Just upload your code in one of the languages that AWS Lambda supports (currently Node.js, Java, C#, Go, and Python) and Lambda takes care of everything required to run and scale your code with high availability. You can set up your code to automatically trigger from other AWS services or call it directly from any web or mobile app. You pay only for the compute time you consume - there is no charge when your code is not running.
```

Comment:
This is exact same content in the AWS Lambda docs. We cannot use the same thing as this will be considered as copyright infringement.

Reply:
Updated.